documentation/modules/ROOT/pages/modules/cdc-mysql-connector/c_mysql-connector-events.adoc

```diff
@@ -351,32 +374,57 @@ CREATE TABLE customers (
       "thread": 7,
       "query": "UPDATE customers SET first_name='Anne Marie' WHERE id=1004"
     },
-    "op": "u", <2>
+    "op": "u",
     "ts_ms": 1465581029523
   }
 }
 ----
 
+Comparing this to the value in the _insert_ event, you can see a couple of differences in the `payload` section:
```

Comment:
I like the callouts + table structure there is for the first example. Perhaps nice to use it consistently for all the event examples? Also can be done separately if you prefer to get this one done quickly.